common.sc

```diff
@@ -21,7 +31,38 @@ trait DiplomaticModule extends ScalaModule {
   // upstream RocketChip in dev branch
   def rocketchipModule: ScalaModule
   override def moduleDeps = super.moduleDeps :+ rocketModule :+ rocketchipModule
-
   override def scalacPluginClasspath = T(super.scalacPluginClasspath() ++ rocketModule.chisel3PluginJar())
   override def scalacOptions = T(super.scalacOptions() ++ rocketModule.chisel3PluginJar().map(p => s"-Xplugin:${p.path}"))
 }
+
+// Build targets to codegen instructions
+// millSourcePath should be set to riscv/riscv-opcodes to generate required Scala file
+trait RiscvOpcodesModule extends Module {
+  // path to script calling python library in riscv/riscv-opcodes.git
+  def script: T[PathRef]
+  def rv64Instruction = T {
+    val f = T.ctx.dest / "Instructions64.scala"
```

Comment:
The reason for separating Instrucitons32.scala is that `python3 parse.py -chisel rv*` will match both `rv32*` and `rv64*` and those instructions with the same name in rv32 will be overriden by those in rv64. So an `Instructions.scala` generated by `rv*` and a `Instructions.scala` generated by `rv32*` would suffice.

Or more concisely, we should generate `Instructions.scala` using `rv_*` instead of `rv*` and `Instructions64.scala` will do its part. I am trying this path.

Reply:
An extra note: current `Instructions.scala` in `rocket-chip` should be generated by `rv* unratified/rv_zicond`, but this is OK for `rocket` now.